mandalab_visual/consts.py

```diff
@@ -0,0 +1,8 @@
+import os
+
+NOTEBOOK_RES_PATH = "/api/mandalab/res/"
+LOCAL_RES_PATH = os.path.join(os.path.dirname(__file__), "res", "") # The last "" is for adding a slash at the end of the path.
```

Comment:
consider using 
`from notebook.utils import url_path_join`
looks like this 
`url_path_join(base_url, 'templates/names')`
maybe this will solve the slash issue with the `""`